tests/Fixtures/Events/PostEvent.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace HeadlessLaravel\Formations\Tests\Fixtures\Events;
+
+use HeadlessLaravel\Formations\Tests\Fixtures\Models\Post;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+class PostEvent
```

Comment:
We can remove this I think since events aren't really needed per the convo we had on the issue

Reply:
Done